src/main/resources/banner.txt

```diff
@@ -1,13 +1,2 @@
-_  _ _  _ ___  ___  ____  
-|__| |\/| |__] |__] [__   
-|  | |  | |    |    ___]  
-                          
-___ ____ _  _ ___  _    ____ ___ ____ 
- |  |___ |\/| |__] |    |__|  |  |___ 
- |  |___ |  | |    |___ |  |  |  |___ 
-                                      
-_  _ ____ ___ _    _ _  _ 
-|_/  |  |  |  |    | |\ | 
-| \_ |__|  |  |___ | | \| 
-                          
-Please change me by generating your own ASCII art and placing in banner.txt
+|__|  |\/| |__) /__` /__`    /  \ |\ | |__     |__) |     /\  |\ |     /\  |__) |
+|  |  |  | |    .__/ .__/    \__/ | \| |___    |    |___ /~~\ | \|    /~~\ |    |
```

Comment:
love this 😁

Reply:
I just realized I got the acronym wrong again! 🤦